---
content_type: page
description: Study materials includes supplementary readings and a sample version
  of the final quiz.
learning_resource_types: []
ocw_type: CourseSection
title: Study Materials
uid: 05cb2fb1-427a-e834-53a1-0c0a64e9766d
video_files:
  video_thumbnail_file: null
video_metadata:
  youtube_id: null
---

This page includes supplementary readings and a sample version of the [final quiz](#smpl_quiz).

References
----------

### Motif Discovery

These reference materials have been compiled with introductory notes by course TA Tim Danford.

Barash, Bejerano, and Friedman. "[A Simple Hyper-Geometric Approach for Discovering Putative Transcription Factor Binding Sites](https://link.springer.com/chapter/10.1007/3-540-44696-6_22)." _Lecture Notes in Computer Science_ 2149 (2001).

Bailey, and Elkan. "The value of prior knowledge in discovering motifs with MEME." _Proc Int Conf Intell Syst Mol Biol_ 3 (1995). ([PS](https://pdfs.semanticscholar.org/67a9/68df4693b00dd1ea5b6fca2bb3933eee61ae.pdf?_ga=2.239756547.1443061890.1563980296-672817412.1563980296))

Neuwald, A. F., J. S. Liu, and C. E. Lawrence. "[Gibbs motif sampling: Detection of bacterial outer membrane protein repeats](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2143180/)." _Protein Science_ 4, no. 8 (1995).

Lawrence, Altschul, Boguski, Liu, Neuwald, and Wootton. "Detecting Subtle Sequence Signals: A Gibbs Sampling Strategy for Multiple Alignment." _Science_ 262, no. 5131 (1993).

Sinha, Saurabh, and Martin Tompa. "[Discovery of novel transcription factor binding sites by statistical overrepresentation](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=140044)." _Nucleic Acids Research_ 30, no. 24 (2002): 5549–5560.

Benos, Panayiotis V., Martha L. Bulyk, and Gary D. Stormo. "[Additivity in protein-DNA interactions: how good an approximation is it?](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=137142)" _Nucleic Acids Res._ 30, no. 20 (2002): 4442–4451.

Bulyk, Martha L., Philip L. F. Johnson, and George M. Church. "[Nucleotides of transcription factor binding sites exert interdependent effects on the binding affinities of transcription factors](http://nar.oxfordjournals.org/cgi/content/abstract/30/5/1255)." _Nucleic Acids Res._ 30, no. 5 (2002): 1255-1261.

Roth, Frederick P., Jason D. Hughes, Preston W. Estep, and George M. Church. "[Finding DNA regulatory motifs within unaligned noncoding sequences clustered by whole-genome mRNA quantitation](http://www.nature.com/cgi-taf/DynaPage.taf?file=/nbt/journal/v16/n10/abs/nbt1098-939.html)." _Nature Biotechnology_ 16 (1998): 939-945.

Hughes, Jason D., Preston W. Estep, Saeed Tavazoie, and George M. Church. "[Computational identification of _Cis_\-regulatory elements associated with groups of functionally related genes in _Saccharomyces cerevisiae_](http://dx.doi.org/10.1006/jmbi.2000.3519)." _Journal of Molecular Biology_ 296, no. 5 (2000): 1205-1214.

Liu, X. Shirley, Douglas L. Brutlag, and Jun S. Liu. "[An algorithm for finding protein-DNA binding sites with applications to chromatin-immunoprecipitation microarray experiments](http://www.nature.com/nbt/journal/v20/n8/abs/nbt717.html)." _Nature Biotechnology_ 20, no. 8 (2002): 835-839.

### Sequence Alignment

These reference materials have been compiled with introductory notes by course TA Tim Danford.

Zhu, Jun, Jun S. Liu, and Charles E. Lawrence. "[Bayesian adaptive sequence alignment algorithms](http://bioinformatics.oxfordjournals.org/content/14/1/25.short)." _Bioinformatics_ 14, no. 1 (1998): 25-39.

Thompson, J. D., D. G. Higgins, and T. J. Gibson. "[CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=308517)." _Nucleic Acids Research_ 22, no. 22 (1994): 4673–4680.

Henikoff, S., and J. G. Henikoff. "[Amino acid substitution matrices from protein blocks](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=50453)." _PNAS_ 89, no. 22 (1992): 10915–10919.

Lipman, D. J., S. F. Altschul, and J. D. Kececioglu. "[A tool for multiple sequence alignment](http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=287279)." _PNAS_ 86, no. 12 (1989): 4412–4415.

### Microarray Normalization and Expression

Newton, M. A., C. M. Kendziorski, C. S. Richmond, F. R. Blattner, and K. W. Tsui. "[On Differential Variability of Expression Ratios: Improving Statistical Inference about Gene Expression Changes from Microarray Data](http://www.liebertonline.com/doi/abs/10.1089%2F106652701300099074)." _Journal of Computational Biology_ 8, no. 1 (2001): 37-52.

Dror, Ron O., Jonathan G. Murnick, Nicola J. Rinaldi, Voichita D. Marinescu, Ryan M. Rifkin, and Richard A. Young. "Bayesian Methods for Transcript Level Estimation from Noisy Array Measurements."

Ideker, Trey, Vesteinn Thorsson, Andrew F. Siegel, and Leroy E. Hood. "[Testing for Differentially-Expressed Genes by Maximum-Likelihood Analysis of Microarray Data](http://www.liebertonline.com/doi/abs/10.1089%2F10665270050514945)." _Journal of Computational Biology_ 7, no. 6 (2000): 805-817.

Hartemink, Alexander J., David K. Gifford, Tommi S. Jaakkola, and Richard A. Young. "Maximum likelihood estimation of optimal scaling factors for expression array normalization." ([PDF - 1.7 MB](https://groups.csail.mit.edu/cgs/pubs/spie.pdf))

Baldi, Pierre, and Anthony D. Long. "[A Bayesian framework for the analysis of microarray expression data: regularized t -test and statistical inferences of gene changes](http://bioinformatics.oxfordjournals.org/cgi/content/abstract/17/6/509)." _Bioinformatics_ 17, no. 6 (2001): 509-519.

Yang, Yee Hwa, Sandrine Dudoit, Percy Luu, David M. Lin, Vivian Peng, John Ngai, and Terence P. Speed. "[Normalization for cDNA microarray data: a robust composite method addressing single and multiple slide systematic variation](http://nar.oxfordjournals.org/cgi/content/abstract/30/4/e15)." _Nucleic Acids Research_ 30, no. 4 (2002): e15.

{{< anchor "smpl_quiz" >}}{{< /anchor >}}Sample Quiz
----------------------------------------------------

Potential version of Final Quiz ({{% resource_link e176b08a-6ece-44f4-cbf3-eecc51aac17f "PDF" %}})